src/storage.ts

```diff
@@ -29,12 +31,13 @@ class IpfsStorageProvider {
       this.settings.port
     }/api/v0/add?pin=true&hash=sha2-256`;
 
-    const formData = new FormData();
-    formData.append("path", msg);
-
+    const formDataBoundary = "2758264728364323843263";
```

Comment:
What is `formDataBoundary`? Where does number come from?

Reply:
Its a string thats unlikely to occur within the data being posted. I just mashed the numpad a bit